src/Wave.hpp

```diff
@@ -0,0 +1,122 @@
+#ifndef GAZEBO_WAVE_PLUGIN_HPP
+#define GAZEBO_WAVE_PLUGIN_HPP
+
+#include "base/Time.hpp"
+#include <gazebo/common/common.hh>
+#include <gazebo/physics/physics.hh>
+#include <gazebo/transport/transport.hh>
+#include <ignition/math/Vector3.hh>
+
+namespace gazebo_usv {
+    class Wave {
+        typedef gazebo::physics::ModelPtr ModelPtr;
+        typedef gazebo::physics::LinkPtr LinkPtr;
+        typedef gazebo::transport::NodePtr NodePtr;
+        typedef gazebo::transport::SubscriberPtr SubscriberPtr;
+
+    public:
+        // Parameters used to calculate the Wave effects
+        struct EffectParameters {
+            double torque_constant;
```

Comment:
What is the constant supposed to be? Rename it or at least provide some detail about it's meaning

Reply:
It was a parameter for setting a scaler for the roll torque, but since we are giving directly the torque value this constant is useless. Deleting it..